api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -34,10 +34,12 @@
 @Table(name = "user")
 public class DbUser {
 
-  private static final String CLUSTER_NAME_PREFIX = "all-of-us-";
+  private static final String RUNTIME_NAME_PREFIX = "all-of-us-";
 
   /**
-   * This is a Gson compatible class for encoding a JSON blob which is stored in MySQL. This
+   * TODO(RW-5406): Remove cluster config overrides.
+   *
+   * <p>This is a Gson compatible class for encoding a JSON blob which is stored in MySQL. This
    * represents cluster configuration overrides we support on a per-user basis for their notebook
```

Comment:
a few missed cluster -> runtime

Reply:
In this case, I haven't bothered renaming these models / comments since they are slotted for deletion, per above RW-5406.